sympy/ntheory/tests/test_factor_.py

```diff
@@ -562,3 +562,28 @@ def test_primeomega():
     assert primeomega(n)
     assert primeomega(n).subs(n, 2 ** 31 - 1) == 1
     assert summation(primeomega(n), (n, 2, 30)) == 59
+
+
+def test_mersenne_prime_exponent():
```

Comment:
Here and in subsequent functions add tests that raise `ValueError` so as to improve code coverage.